content/faq.md

```diff
@@ -84,4 +84,4 @@ title = "Frequently Asked Questions (FAQ)"
 
 * Is there any way to boost adventure AI? I want harder opponent.
 
-> Yeah, the easiest thing to try is to use cheats for resources and/or full map reveal, using parameter that specifies target players. Typing 'ai' after cheat name causes only AI players to be affected by it, player color names are also recognized. So you can for example type 'vcmiformenos ai' to give extra resources to AI player, 'vcmieagles blue' to reveal full map for blue player etc.  Note that these cheats do not guarantee any improvement, for example revealing whole map may cause AI to overly focus on far goals, missing local details. Distance is one of factors when deciding what to do, but still. More advanced thing you can do is experimenting with AI Values for various objects / object classes by changing AI Value (exact object value) and Default AI Value (object class value) in object config JSON files. Range of effective AI Value is 0-20000. Also it's important to know that AI Values are tied to saved games by design, so you need to start new game to test effectiveness of new AI Values.
\ No newline at end of file
+> Yeah, the easiest thing to try is to use cheats for resources and/or full map reveal, using parameter that specifies target players. Typing 'ai' after cheat name causes only AI players to be affected by it, player color names are also recognized. So you can for example type 'vcmiformenos ai' to give extra resources to AI player, 'vcmieagles blue' to reveal full map for blue player etc.  Note that these cheats do not guarantee any improvement, for example revealing whole map may cause AI to overly focus on far goals, missing local details. Distance is one of factors when deciding what to do, but still. More advanced thing you can do is experimenting with AI Values for various objects / object classes by changing AI Value (exact object value) and Default AI Value (object class value) in object config JSON files. Range of effective AI Value is 0-20000. Also it's important to know that AI Values are tied to saved games by design, so you need to start new game to test effectiveness of new AI Values.
```

Comment:
Don't see the difference in this text

Reply:
Resourceful AI mod may be mentioned here.

Reply:
I'm not sure how that got changed. Maybe it's a line ending change? I don't remember changing that line.

Reply:
It likely end of line symbol changed.